styles.css

```diff
@@ -89,11 +122,635 @@
+  padding-left: 12px;
+  margin-top: 20px;
+}
+
+.nav-list {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .grid-container {
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+    margin: 50px;
+  }
+
+  .header-bar {
+    display: none;
+  }
+
+  .header .header-bar {
+    display: none;
+  }
+
+  .bank {
+    background-image: url("./images/img\ Placeholderdesek2.png");
+    background-repeat: no-repeat;
+    opacity: 10;
+    background-size: cover;
+    font-family: sans-serif;
+    font-style: normal;
+    font-weight: 100;
+    font-size: 20px;
+    line-height: 44px;
+    color: #fff;
+  }
+
+  .project-card {
+    height: 400px;
+  }
+
+  .recent-work {
+    display: grid;
+    grid-template-columns: 30% 70%;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .list {
+    justify-content: flex-start;
+    gap: 5px;
+  }
+
+  .nav-list {
+    display: flex;
+    margin-right: 15px;
+  }
+
+  .nav-list li {
+    padding: 0 10px;
+  }
+
+  .nav-list a {
+    color: #fff;
+  }
+
+  .main-section > div {
+    order: 2;
+  }
+
+  .main-section > ul {
+    order: 1;
+    flex-direction: column;
+    padding: 5px;
+    margin-right: 10px;
+    width: fit-content;
+    justify-content: flex-start;
+  }
+
+  .project {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 50px;
+    box-sizing: border-box;
+  }
+
+  .project li > div {
+    display: flex;
+    height: 100%;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .project-text {
+    font-family: 'Inter', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 24px;
+    color: #fff;
+  }
+
+  .rmv {
+    display: none;
+  }
+
+  #bvn { display: none; }
+
+  .card-button {
+    padding: 12px;
+    width: 100%;
+    font-size: 15px;
+    margin-top: 100%;
+  }
+
+  .card-button:enabled {
+    background: #f55800;
+    color: #fff;
+    border: none;
+  }
+
+  .card-button:disabled {
+    background: #e5e5e5;
+    border: none;
+    color: #5b5957;
+  }
+
+  .card-button:hover {
+    border: 1px solid #f55800;
+    background: #fff5e1;
+    color: #f55800;
+  }
+
+  .header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 15px;
+  }
+
+  .header-bar ul {
+    display: flex;
+    flex-direction: row;
+    margin-right: 30px;
+  }
+
+  .nav-head {
+    padding-right: 5rem;
+    margin-right: 20px;
+  }
+
+  .nav-head img {
+    margin-left: 13px;
+  }
+
+  .nav-head li a {
+    display: flex;
+    flex-direction: row;
+    margin-right: 12;
+    text-decoration: none;
+    color: whitesmoke;
+  }
+
+  .main-section {
+    display: flex;
+    background-image: url("./images/Illustrationheader.png");
+    background-repeat: no-repeat;
+    background-position: right -1% bottom 22%;
+    background-size: 511px;
+    background-color: #1c1a19;
+    height: 94vh;
+    padding: 100px 27px 0;
+  }
```

Comment:
Your desktop main section doesn't look quite like the one from the template design.

Here's your design:

<img width="1440" alt="Screen Shot 2022-05-30 at 15 07 26" src="https://user-images.githubusercontent.com/82426682/171054191-1f1ae53e-b8d5-4d93-a393-c03ab2d47171.png">

Here's the template design:
<img width="778" alt="Screen Shot 2022-05-30 at 15 08 01" src="https://user-images.githubusercontent.com/82426682/171054259-6172687c-55d8-4539-8691-f3a2fcc5724c.png">

Do you see how different they look from one another? Please follow the template design closer. Here are some things to think about: 

- Margins
- Paddings
- Background image sizes
- Font style and font size
- Icons